test/e2e/network_test.go

```diff
@@ -25,10 +25,10 @@ func testHNSNetworksCreated(t *testing.T) {
 		// network setup has succeeded.
 		stdout, _, err := vm.Run("Get-HnsNetwork", true)
 		require.NoError(t, err, "Could not run Get-HnsNetwork command")
-		assert.Contains(t, stdout, "BaseOpenShiftNetwork",
-			"Could not find BaseOpenShiftNetwork in %s", vm.GetCredentials().GetInstanceId())
-		assert.Contains(t, stdout, "OpenShiftNetwork",
-			"Could not find OpenShiftNetwork in %s", vm.GetCredentials().GetInstanceId())
+		assert.Contains(t, stdout, "BaseOVNKubernetesHybridOverlayNetwork",
+			"Could not find BaseOVNKubernetesHybridOverlayNetwork in %s", vm.GetCredentials().GetInstanceId())
+		assert.Contains(t, stdout, "OVNKubernetesHybridOverlayNetwork",
+			"Could not find OVNKubernetesHybridOverlayNetwork in %s", vm.GetCredentials().GetInstanceId())
```

Comment:
This needs to be fixed in the main code too at https://github.com/openshift/windows-machine-config-operator/pull/38/commits/f9f66a33ac48967de5e845eff6419048dacfcff2#diff-5126e46384aa6511918e3e668e4ffa5eR153. Please make these constants while doing so.

Reply:
I don't understand how the e2e is passing :confused:

Reply:
Thanks for catching it, I looked at the logs, https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-operator/38/pull-ci-openshift-windows-machine-config-operator-master-e2e-operator/205/artifacts/e2e-operator/container-logs/test.log

This is actually throwing an error but we're suppressing the error [here](https://github.com/openshift/windows-machine-config-operator/blob/92131b00410d1bf341b967571555c593b9dcb565/pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go#L276)

Reply:
And this should be addressed by https://github.com/openshift/windows-machine-config-operator/pull/35/files#diff-c9e016282d2886f8f7fc2355cc2c1bf4R236